{black_filter_enabled: false, black_hue: 0.0, black_hue_variance: 0.0, black_sat: 255.0,
  black_sat_variance: 0.0, black_threshold: 0.0, black_val: 255.0, black_val_variance: 0.0,
  blue_filter_enabled: false, blue_hue: 120.0, blue_hue_variance: 0.0, blue_sat: 255.0,
  blue_sat_variance: 0.0, blue_threshold: 0.0, blue_val: 255.0, blue_val_variance: 0.0,
  close_iterations: 1, enable_meanshift: true, enable_thresholding: false, green_filter_enabled: false,
  green_hue: 84.571914672851562, green_hue_variance: 13.145003318786621, green_sat: 174.00978088378906,
  green_sat_variance: 111.38517761230469, green_threshold: 0.0030000000000000001,
  green_val: 236.62034606933594, green_val_variance: 815.34832763671875, ms_color_radius: 40.0,
  ms_color_ratius: 40.0, ms_max_iter: 2, ms_max_level: 2, ms_min_epsilon: 1.0, ms_spatial_radius: 10.0,
  open_iterations: 1, orange_filter_enabled: false, orange_hue: 40.848214285714285,
  orange_hue_variance: 1211.2766569368014, orange_sat: 18.997252747252748, orange_sat_variance: 360.60164080425051,
  orange_threshold: 1.0000000000000001e-05, orange_val: 235.54189560439559, orange_val_variance: 1094.8526403627272,
  red_filter_enabled: false, red_hue: 0.0, red_hue_variance: 0.0, red_sat: 255.0,
  red_sat_variance: 0.0, red_threshold: 0.0, red_val: 255.0, red_val_variance: 0.0,
  threshold_max_value: 0.0, threshold_type: 1, threshold_value: 0.0, white_filter_enabled: false,
  white_hue: 0.0, white_hue_variance: 0.0, white_sat: 255.0, white_sat_variance: 0.0,
  white_threshold: 0.0, white_val: 255.0, white_val_variance: 0.0, yellow_filter_enabled: false,
  yellow_hue: 30.0, yellow_hue_variance: 0.0, yellow_sat: 255.0, yellow_sat_variance: 0.0,
  yellow_threshold: 0.0, yellow_val: 255.0, yellow_val_variance: 0.0}
